openfisca_us/variables/hhs/hhs_smi.py

```diff
@@ -0,0 +1,29 @@
+from openfisca_core.model_api import *
+
+
+class hhs_smi(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = u"State Median Income (DHHS)"
```

Comment:
I don't know why, but HHS isn't referred to with a D
```suggestion
    label = u"State Median Income (HHS)"
```